recipes-ni/ni-utils/files/x64/fw_printenv

```diff
@@ -235,7 +235,7 @@ parse_opt(){
 
 	#check if there are other options passed incorrectly
 	shift $(($OPTIND - 1))
-	if [ "x$(echo $* | grep ' \-')" != "x" ]; then
+	if [ "x$(echo $* | grep ' -')" != "x" ]; then
```

Comment:
I assume you already tested that the behavior of new grep without the escape character is same as old grep with escape character?